homeassistant/components/alexa/entities.py

```diff
@@ -645,6 +653,26 @@ def interfaces(self) -> Generator[AlexaCapability]:
         yield Alexa(self.entity)
 
 
+@ENTITY_ADAPTERS.register(remote.DOMAIN)
+class RemoteCapabilities(AlexaEntity):
+    """Class to represent Remote capabilities."""
+
+    def default_display_categories(self) -> list[str]:
+        """Return the display categories for this entity."""
+        return [DisplayCategory.TV]
```

Comment:
A TV does not reflect a remote. The existing `media_player` entity would be a better match.

Reply:
Alexa supports media player entities to control your TV: https://www.home-assistant.io/integrations/alexa.smart_home/#media-player

Reply:
You are right. There even exists a remote display category for alexa (https://developer.amazon.com/en-US/docs/alexa/device-apis/alexa-discovery.html#display-categories).
I added the remote display category and used it for the RemoteCapabilities.